### ISSUE TITLE
Implement a 'fake' createReadStream

### ISSUE THREAD

**Issue by kdvolder** [eldargab/node-fake-fs#6]
Fake-fs as it is, is already quite useful. It would be even more useful if it implemented more of the node fs api. 

One of the missing functions is 'createReadStream'.

As a first step it might already be useful to provide a mock implementation that emulates the Stream like interaction without truly needing to support true Streaming. 

For example something similar to this:

https://github.com/scripted-editor/scripted/blob/dev/server/plugable-fs/scripted-fs.js#L367

**Comment by eldargab** [comment]
Yes, this is not beyond the scope. We can add it after new stream interface will be released. Don't think it is wise to bother with it right now

**Comment by Olegas** [comment]
@kdvolder take a look at https://github.com/Olegas/node-mockfs
